### DISCUSSION TITLE
Norman Layout

### DISCUSSION THREAD

**Discussion by dunyakirkali** [manna-harbour/miryoku#257] [Q&A] [answered]
Is there a guide I can follow to add the [Norman layout](https://normanlayout.info/index.html) to miryoku?

**Comment by manna-harbour** [comment, marked as answer]
You can substitute a custom Base layer as described at https://github.com/manna-harbour/miryoku/discussions/85.

**Reply by dunyakirkali** [reply]
Awesome! Thank you
